code/game/machinery/iv_drip.dm

```diff
@@ -57,6 +83,7 @@
 			"You attach \the [src] to \the [over_object].")
```

Comment:
Its out of scope for this PR, but it is kinda weird that the IV drip doesn't have a skill check like the blood bag does.